pkg/sql/opt/exec/execbuilder/testdata/show_trace

```diff
@@ -197,10 +197,9 @@ SET tracing = off
 query TT
 $trace_query
 ----
-drop index  Del /Table/24/1/3/108/2/0
```

Comment:
What was this del?

Reply:
This was to clean up the comment for the index, this was I think always done before versus the declarative world where we detect if one exists during the build phase. We pre-scan the first 1000 comments inside the collections if I remember correctly.